library/registry

```diff
@@ -1,12 +1,12 @@
-# maintainer: Joffrey F <joffrey@docker.com> (@shin-)
-# maintainer: Sam Alba <sam@dotcloud.com> (@samalba)
 # maintainer: Stephen Day <stephen.day@docker.com> (@stevvooe)
 # maintainer: Olivier Gambier <olivier.gambier@docker.com> (@dmp42)
+# maintainer: Richard Scothern <richard.scothern@gmail.com>
 
 latest: git://github.com/docker/docker-registry@0.9.1
 0.8.1: git://github.com/docker/docker-registry@0.8.1
 0.9.1: git://github.com/docker/docker-registry@0.9.1
 
-2: git://github.com/docker/distribution@62b70f951f30a711a8a81df1865d0afeeaaa0169
-2.0: git://github.com/docker/distribution@62b70f951f30a711a8a81df1865d0afeeaaa0169
+2: git://github.com/docker/distribution@1341222284b3a6b4e77fb64571ad423ed58b0d34
+2.0: git://github.com/docker/distribution@1341222284b3a6b4e77fb64571ad423ed58b0d34
 2.0.0: git://github.com/docker/distribution@62b70f951f30a711a8a81df1865d0afeeaaa0169
```

Comment:
It's an odd problem, too.  This file isn't the release artifact, it's just the instructions to the "release artifact builder" for what release artifacts to build.  The actual release artifact is the built image, so those are tracked completely via tags on the Hub.

See tags listed on https://registry.hub.docker.com/_/redis/ vs https://registry.hub.docker.com/_/redis/tags/manage/ for a good concrete example of this.

I'm not seeing this documented well in our `README.md` though, so that's definitely something we need to rectify (it's implied by phrases like `The generated image will be tagged as <manifest-filename>:<docker-tag>.`, but that's not nearly explicit enough to make it obvious). :+1: